CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-## vUNRELEASED
+## v0.12.0
 
-The next version of rkt will have the following.
+rkt v0.12.0 is an incremental release with user-facing changes, bug fixes and testing improvements.
```

Comment:
How about:

rkt v0.12.0 is an incremental release with UX improvements like fine-grained security controls and implicit generation of empty volumes, performance improvements, bug fixes and testing enhancements.